lib/FilePathResolver/Tests/Unit/Expander/CallbackExpanderTest.php

```diff
@@ -19,14 +18,4 @@ public function testExpandsCallbackValue(): void
     {
         $this->assertEquals('bar', $this->expand('%foo%'));
     }
-
-    public function testThrowsExeceptionWhenCallbackReturnsNonString(): void
```

Comment:
We could replace this test with a `TypeError` exception. But at this point we're testing php works and phpstan is not going to like this.